Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -1080,9 +1080,6 @@ - (nonnull UICollectionReusableView *)collectionView:(UICollectionView *)collect
             return [collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:WMFFeedEmptyHeaderFooterReuseIdentifier forIndexPath:indexPath];
         case WMFFeedMoreTypeLocationAuthorization: {
             WMFTitledExploreSectionFooter *footer = (id)[collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:[WMFTitledExploreSectionFooter wmf_nibName] forIndexPath:indexPath];
-            footer.titleLabel.text = [EnableLocationViewController localizedEnableLocationTitle];
-            footer.descriptionLabel.text = [EnableLocationViewController localizedEnableLocationDescription];
-            [footer.enableLocationButton setTitle:[EnableLocationViewController localizedEnableLocationButtonTitle] forState: UIControlStateNormal];
```

Comment:
@nambatee that sounds good - I messed that up in my PR, I didn't realize it was just "Explore articles near your location" instead of "Explore articles near your location by enabling Location Access"